app/assets/src/components/ui/icons/HeatmapPrivate.jsx

```diff
@@ -0,0 +1,359 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import cs from "./heatmap_private.scss";
+
+/**
+ * This class was created by the following process:
+ * Extracted CSS to heatmap_private.scss.
+ * From Adobe generated SVG to JSX using https://magic.reactjs.net/htmltojsx.htm.
+ * Transformed CSS classnames.
+ */
```

Comment:
These SVGs can be simplified significantly.
I can do that in a follow up PR.

Reply:
As an aside here is a PR that simplified some icons (https://github.com/chanzuckerberg/idseq-web/pull/2040).

Reply:
Ok thanks for pointing me to that. It's not obvious to me how to know what to remove without knowing the semantics of SVG. 

Is it worth the effort in general to hand-edit SVG? There must be a standard way to export SVGs that doesn't require that much manual labor.

Reply:
Yay for the icons!

@gregdingle Are you still working on updating the titles?

Reply:
@happyimadesignr The titles are now simply the name of the phylo tree or "heatmap", as we discussed, though that's not obvious from the screen I posted.

Reply:
Cool, got it!